package.json

```diff
@@ -47,22 +47,24 @@
     "eslint-plugin-standard": "^3.0.1",
     "jest": "^19.0.2",
     "prettier": "^1.3.1",
-    "react": "^15.4.2",
-    "react-dom": "^15.4.2",
-    "react-test-renderer": "^15.4.2",
+    "react": "^15.5.0",
+    "react-dom": "^15.5.0",
+    "react-test-renderer": "^15.5.0",
     "semantic-release": "^6.3.6"
   },
   "peerDependencies": {
-    "react": "^15.0.0",
-    "patternfly": "~3.3"
+    "classnames": "^2.2.5",
+    "immutability-helper": "^2.2.0",
+    "prop-types": "^15.5.10",
+    "react": "^15.5.0"
   },
   "scripts": {
     "commit": "git-cz",
     "build": "babel src --out-dir lib --ignore test.js",
```

Comment:
It should work, in 'build' task storyboard is not built as storyboard is not included in production lib directory.